app/views/avo/fields/edit/_select-field.html.erb

```diff
@@ -0,0 +1,23 @@
+<%
+  if !field[:options].nil?
```

Comment:
This whole `if/else` is very difficult to understand. Also I think it doesn't really work in all situations. 

The reason is that if it's an `enum` we never detect it. And because we don't detect it we'll never know if the `value` is truly the value. Might be the `label` and the `enum` casted it.

Example:

If in the DB we have `done` when we configure the enum, we'll always get the label `Done`. If it's just an `options` hash we'll get the raw value `done`.
If we get the label `Done` we won't be able to treat that.

I'd bring back the `enum` option on the field.

Reply:
I've come up with something that works. Let me know when you review in order to look on it together. There's no need for both enum and options.